traitsui/key_bindings.py

```diff
@@ -294,3 +260,48 @@ def _match_binding(self, binding, skip=frozenset()):
             for a_binding in self.bindings
             if a_binding not in skip and a_binding.match(binding)
         )
+
+
+class KeyBindingsHandler(ModelView):
+
+    bindings = List(Instance(KeyBinding))
+
+    def key_binding_for(self, binding, key_name):
+        """Returns the current binding for a specified key (if any)."""
+        if key_name != "":
+            for a_binding in self._match_binding(key_name, skip={binding}):
+                return a_binding
+
+        return None
+
+    def _match_binding(self, binding, skip=frozenset()):
+        """Return all KeyBinding instances that match the given binding.
+        """
+        return (
+            a_binding
+            for a_binding in self.bindings
+            if a_binding not in skip and a_binding.match(binding)
+        )
```

Comment:
do we need the duplication here? or can we simply call the method on the model of the same name?

Reply:
It's needed because the `bindings` list on this object is sorted differently from the one on the underlying model.